docs/book/src/clusterctl/configuration.md

```diff
@@ -160,9 +160,16 @@ all the images in the cert-manager component.
 For situations when resources are limited or the network is slow, the cert-manager wait time to be running can be customized by adding a field to the clusterctl config file, for example:
 
 ```yaml
-  cert-manager-timeout: 15m 
+  cert-manager-timeout: 15m
 ```
 
 The value string is a possibly signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 
-If no value is specified or the format is invalid, the default value of 10 minutes will be used.
\ No newline at end of file
+If no value is specified or the format is invalid, the default value of 10 minutes will be used.
+
+
+## Debugging/Logging
+
+To have more verbose logs you can use the `-v` flag when running the `clusterctl` and set the level of the logging verbose from 0 to 5.
+
+If you do not want to use the flag every time you issue a command you can set the environment variable `CLUSTERCTL_LOG_LEVEL` or set the variable in the `clusterctl` config file which is located by default at `$HOME/.cluster-api/clusterctl.yaml`.
```

Comment:
It may be helpful to have a link to the book at https://cluster-api.sigs.k8s.io/clusterctl/configuration.html for more details.

Reply:
not clear to me your comment, this piece of the documentation will be  in the https://cluster-api.sigs.k8s.io/clusterctl/configuration.html page

Reply:
Fair point, I should have paid attention to the path of the file. :sweat_smile: